src/main/java/frc/states/MechStates/Intook.java

```diff
@@ -0,0 +1,79 @@
+package frc.states.MechStates;
+import frc.robot.commands.DefaultMechCommand;
+import frc.robot.constants.LauncherConstants.LauncherMode;
+import frc.robot.subsystems.AimingSubsystem;
+
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.LauncherSubsystem;
+import frc.states.MechState;
+
+public class Intook implements MechState {
+
+    DefaultMechCommand mMechCommand;
+    
+    IntakeSubsystem mIntakeSubsystem;
```

Comment:
Shoud these fields be `private final`?

Reply:
This same comment applies to the other types too in the PR.

Reply:
done, all member variables are either private final or private